README.md

```diff
@@ -118,25 +118,59 @@ the steps.
 
 ### Echidna
 
-First install [Echidna]. 
+First install [Echidna].
 
 Examples:
+
 ```
 $ echidna-test . --contract Echidna_Valid_Deposit --config contracts/echidna/Echidna_Deposit_Withdraw.yml
 ```
+
 In order to initialize echidna install [Etheno] and run:
+
 ```
 $ etheno --ganache --ganache-args "--gasLimit=0x1fffffffffffff --allowUnlimitedContractSize -e 1000000000" -x ./init.json --debug
 ```
+
 In another terminal run one test, for example:
+
 ```
 $ yarn hardhat test test/Vault.spec.ts  --grep "works with valid parameters" --network etheno
 ```
+
 Then Ctrl-C in the first terminal (twice) to save.
 
 [echidna]: https://github.com/crytic/echidna
 [etheno]: https://github.com/crytic/etheno
 
+### [Tenderly](https://tenderly.co/)
+
+We are also using Tenderly's Visual Debugger to debug local transactions efficiently.
+
+First you need to create a [tenderly account](https://dashboard.tenderly.co/register).
+
+Then if not installed already, [install tenderly-cli](https://github.com/Tenderly/tenderly-cli#installation) and log in.
+
+To debug any transaction, you need to be inside the docker folder first.

```

Comment:
can we maybe have a bin/script to perform these?

Reply:
@0xrin1 can answer this better

Reply:
ahahah I meant to say that can you please write a shell script that does these steps taking the tx hash an argument?

Reply:
nope, it's good!